types/components/buttons.d.ts

```diff
@@ -108,6 +108,81 @@ export type OnShippingChangeActions = {
     };
 };
 
+type CurrencyCodeAndValue = {
+    currencyCode: string;
+    value: string;
+};
```

Comment:
there is a similar type here: https://github.com/paypal/paypal-js/blob/aa10eff7be5b7cad8d3bd0b446ceec29fcfd736a/types/apis/commons.d.ts#L1-L12 but I wasn't sure if this new type should be added there, as it is not shared with any other types

Reply:
I think you're good here on defining a separate `type`, since it's not intended to be extended or used as a class contract like with an `interface`.